cmd/commandfuncs.go

```diff
@@ -608,32 +601,13 @@ func cmdUpgrade(_ Flags) (int, error) {
 	for pkg := range pluginPkgs {
 		qs.Add("p", pkg)
 	}
-	urlStr := fmt.Sprintf("https://caddyserver.com/api/download?%s", qs.Encode())
 
 	// initiate the build
-	l.Info("requesting build",
-		zap.String("os", qs.Get("os")),
-		zap.String("arch", qs.Get("arch")),
-		zap.Strings("packages", qs["p"]))
-	resp, err := http.Get(urlStr)
+	resp, err := downloadModule(qs)
```

Comment:
I think this should be `downloadBuild` since we're requesting a new build from the download server.